lib/flutter_vlc_player.dart

```diff
@@ -343,44 +421,206 @@ class VlcPlayerController {
     _onInit();
   }
 
-  Future<void> setStreamUrl(String url) async {
+  Future<void> setStreamUrl(String url,
+      {bool isLocalMedia,
+      String subtitle,
+      bool isLocalSubtitle,
+      bool isSubtitleSelected}) async {
     _initialized = false;
     _fireEventHandlers();
 
-    bool wasPlaying = _playingState != PlayingState.STOPPED;
-    await _methodChannel.invokeMethod("changeURL", {'url': url});
+    bool wasPlaying = (_playingState == PlayingState.PLAYING);
+    await _methodChannel.invokeMethod("changeURL", {
+      'url': url,
+      'isLocalMedia': isLocalMedia ?? false,
+      'subtitle': subtitle ?? '',
+      'isLocalSubtitle': isLocalSubtitle ?? false,
+      'isSubtitleSelected': isSubtitleSelected ?? true,
+    });
     if (wasPlaying) play();
 
     _initialized = true;
     _fireEventHandlers();
   }
 
   Future<void> play() async {
-    await _methodChannel
-        .invokeMethod("setPlaybackState", {'playbackState': 'play'});
+    await _methodChannel.invokeMethod("setPlaybackState", {
+      'playbackState': 'play',
+    });
   }
 
   Future<void> pause() async {
-    await _methodChannel
-        .invokeMethod("setPlaybackState", {'playbackState': 'pause'});
+    await _methodChannel.invokeMethod("setPlaybackState", {
+      'playbackState': 'pause',
+    });
   }
 
   Future<void> stop() async {
-    await _methodChannel
-        .invokeMethod("setPlaybackState", {'playbackState': 'stop'});
+    await _methodChannel.invokeMethod("setPlaybackState", {
+      'playbackState': 'stop',
+    });
+  }
+
+  Future<bool> isPlaying() async {
+    var result = await _methodChannel.invokeMethod("isPlaying");
+    return result;
   }
 
   Future<void> setTime(int time) async {
-    await _methodChannel.invokeMethod("setTime", {'time': time.toString()});
+    await _methodChannel.invokeMethod("setTime", {
+      'time': time.toString(),
+    });
+  }
+
+  Future<int> getTime() async {
+    var result = await _methodChannel.invokeMethod("getTime");
+    return result;
+  }
+
+  Future<int> getDuration() async {
+    var result = await _methodChannel.invokeMethod("getDuration");
+    return result;
   }
 
   Future<void> setVolume(int volume) async {
-    await _methodChannel.invokeMethod("setVolume", {'volume': volume});
+    await _methodChannel.invokeMethod("setVolume", {
+      'volume': volume,
+    });
+  }
+
+  Future<int> getVolume() async {
```

Comment:
Could you please add more comments to the public methods here? As on the pub.dev it's giving lower scores for the package and it's generally would be very helpful for the users.
<img width="726" alt="image" src="https://user-images.githubusercontent.com/40825054/93743111-17eb9e00-fbf8-11ea-8686-6a93dd3e883f.png">